java-frontend/src/main/java/org/sonar/java/model/JProblem.java

```diff
@@ -49,6 +49,18 @@ public String message() {
 
   @Override
   public String toString() {
+    if (type == JProblem.Type.PREVIEW_FEATURE_USED) {
+      return cleanMessage(message);
+    }
+    return message;
+  }
+
+  private static String cleanMessage(String message) {
+    String enablePreviewMessage = " Use --enable-preview to enable";
```

Comment:
We should probably set this string as a constant of the class rather than a variable as it is not likely to change